dev-docs/RFCs/v7.x/data-loading-rfc.md

```diff
@@ -0,0 +1,81 @@
+# RFC: Data Loading Enhancements (for loaders.gl)
+
+* **Author**: Ib Green
+* **Date**: May 2019
+* **Status**: **Draft**
+
+## Abstract
+
+This RFC suggests a number of improvements to loading support in deck.gl, in particular to support seamless optional integration with loaders.gl.
+
+## Background
+
+The intention of loaders.gl (loaders are designed to be directly compatible with luma.gl and deck.gl) is not fully realized in deck.gl v7.0.
+
+
+## Proposal: Add `parse` method
+
+```js
+import {parse, registerLoaders} from '@loaders.gl/core';
+import {CSVLoader} from '@loaders.gl/CSV';
+registerLoaders(CSVLoader);
+
+new AnyLayer({
+  coordinateSystem: COORDINATE_SYSTEM.IDENTITY,
+  data: CSV_URL,
+  // NEW: Accept a parse methods
+  parse,
```

Comment:
How does this work with the current `dataTransform` prop?

Reply:
> How does this work with the current dataTransform prop?

`dataTransform` is still applied to the output of `parse`. 

I will clarify in docs.

Reply:
Arguably you can already achieve this by supplying your own `fetch` prop. What is the input to the `parse` callback? Do we really need separate props for all these stages? Should they be named in a way that imply they are in the same group?